app/assets/stylesheets/shared/smart_annotation.scss

```diff
@@ -36,11 +36,19 @@
     }
 
     .repository-object {
-      max-width: 250px;
       overflow: hidden;
       text-overflow: ellipsis;
       white-space: nowrap;
     }
+
+    .truncate {
```

Comment:
It is tailwind class why you override this?

Reply:
i just named it truncate out of its relevance, no ties to tailwind truncate class, it doesnt yield the desired result, should i rename this one or let it be like this?

Reply:
Why just don't use only tailwind classes?